service/history/queueProcessor.go

```diff
@@ -156,10 +160,10 @@ processorPumpLoop:
 			p.processBatch(tasksCh)
 		case <-pollTimer.C:
 			p.processBatch(tasksCh)
-			pollTimer.Reset(p.options.MaxPollInterval)
+			pollTimer.Reset(jitter.JitDuration(p.options.MaxPollInterval(), p.options.MaxPollIntervalJitterCoefficient()))
```

Comment:
I think we want same logic to skip the forced scan if already performed recently.